marathon-bigip-ctlr.py

```diff
@@ -968,17 +1138,32 @@ def parse_args():
     # parse args
     args = parse_args()
 
-    bigip = CloudBigIP(args.host, args.port, args.username,
-                       args.password, args.partition, token="tmos")
+    # Setup logging
+    setup_logging(logging.getLogger(), args.log_format, args.log_level)
+
+    # BIG-IP to manage
+    bigip = ManagementRoot(
+        args.host,
+        args.username,
+        args.password,
+        port=args.port,
+        token="tmos")
+
+    # Management for the BIG-IP partitions
+    cccls = []
+    for partition in args.partition:
+        cccl = F5CloudServiceManager(
+            bigip,
+            partition,
+            prefix="",
+            schema_path="./src/f5-cccl/f5_cccl/schemas/cccl-api-schema.yml")
+        cccls.append(cccl)
```

Comment:
I don't do anything more than create them and iterate through them, so I don'r see that as necessary.